app/file-detail/controller.js

```diff
@@ -69,10 +83,60 @@ export default Controller.extend({
             window.location = url;
         },
 
-        changeView() {
-            $('#mfrIframeParent').toggle();
-            $('#revisionsPanel').toggle();
-            $('.view-button').toggleClass('btn-default btn-primary');
+        delete() {
+            this.get('model.file').destroyRecord()
+                .then(() => this._deleteFile('success'))
+                .catch(() => this._deleteFile('error'));
+        },
+
+        showModal() {
```

Comment:
should be more specific and use consistent language, e.g. `openDeleteModal()` or maybe even `openDeleteConfirmationModal()`